doc/source/contributing.rst

```diff
@@ -221,14 +218,12 @@ You'll need to have at least python3.5 installed on your system.
    . ~/virtualenvs/pandas-dev/bin/activate
 
    # Install the build dependencies
-   python -m pip install -r ci/requirements_dev.txt
+   python -m pip install -r requirements.txt
```

Comment:
this needs to be updated.

I would call it ``requirements-dev.txt`` (I *think* we are consistent on the ``-`` vs ``_``)